### DISCUSSION TITLE
Force rediscovery in Home Assistant?

### DISCUSSION THREAD

**Discussion by some-rando** [dawidchyrzynski/arduino-home-assistant#39] [Q&A] [answered]
First of all, great library! It's made my little Arduino/Home Assistant project WAY easier. Thanks!

I made the mistake of deleting the auto-discovered device I made from the Home Assistant MQTT integration configuration. Now I can't seem to get the Arduino and it's HABinarySensor(s) and HASensor(s) to be discovered again. My googling has only led me to people saying things like esphome should automatically be rediscovered - leading me to wonder if it's up to the client/Arduino side to broadcast some sort of magical discovery mqtt message rather than Home Assistant. I'm barely knowledgable in MQTT though, so I'm at a loss.

So is there a way to have Home Assistant rediscover my Arduino? I've tried restarting Home Assistant multiple times (a docker container), as well as Mosquitto (mqtt server in another docker container).

**Comment by srainsdon** [comment]
I reset the device and it readded it

**Comment by dawidchyrzynski** [comment, marked as answer]
This feature will be available in the coming release - see #81

**Comment by mickehallgren** [comment]
I'm in the same position as @some-rando . I am using 2.0.0 of ArduinoHA.
And I can't get it to rediscover the device and entities. Do you know of any workaround?
